.devcontainer/README.md

```diff
@@ -13,14 +13,18 @@ https://github.com/microsoft/vscode-dev-containers/tree/v0.140.1/containers/pupp
 
 	// Set *default* container specific settings.json values on container create.
 	"settings": {
-		"terminal.integrated.shell.linux": "/bin/bash"
+		"terminal.integrated.profiles.linux": {
+			"bash": {
+				"path": "bash",
```

Comment:
Codacy has a fix for the issue: [Column: 1](https://app.codacy.com/gh/example42/puppet-psick_profile/pullRequest?prid=9480555)
```suggestion
    "path": "bash",
```